lib/annotate.rb

```diff
@@ -76,6 +76,10 @@ def self.setup_options(options = {})
       options[:model_dir] = ['app/models']
     end
 
+    if(options[:root_dir].empty?)
+      options[:root_dir] = ['/']
```

Comment:
+ Thats true, we could just make the default `''`
+ The place where root_dir is used in will be unchanged:
```
File.join(root_directory, UNIT_TEST_DIR,  "%MODEL_NAME%_test.rb")
```